frontend/src/lib/components/flows/CopyFirstStepSchema.svelte

```diff
@@ -0,0 +1,11 @@
+<script lang="ts">
+	import { copyFirstStepSchema, isCopyFirstStepSchemaDisabled } from './flowStore'
+</script>
+
+<button
+	class="default-secondary-button-v2"
+	disabled={$isCopyFirstStepSchemaDisabled}
+	on:click={async () => await copyFirstStepSchema()}
```

Comment:
`on:click={copyFirstStepSchema}`

no need to async await if no statement afterwards

Reply:
Done